core/src/modules/DynamicsModule/module.cfg

```diff
@@ -21,3 +21,8 @@ has_help = false
 file_prefix = BBMDynamics
 description = Discontinuous Galerkin dynamics with BBM rheology.
 has_help = false
+
+[Nextsim::FreeDriftDynamics]
+file_prefix = FreeDriftDynamics
+description = Free drift dynamics.
+has_help = false
```

Comment:
Just out of curiosity: What is the "has help" field for, what does it define/do?

Reply:
Within a Module, there is a function that fetches the help text from any implementation that has configuration options (this is the text that is shown when running `./nextsim --help-config`). When I was writing the Module classes by hand, I had to choose which of the implementation sub-classes were interrogated to get help text. This field in the module configuration determines the same thing for the scripts that are building the `Module` classes.
There may be a better way to do this, but that should be another issue & PR.